packages/react/src/components/Radio/index.css

```diff
@@ -0,0 +1,112 @@
+.charcoal-radio {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-gap: 4px;
+  align-items: center;
+  cursor: pointer;
+}
+.charcoal-radio[aria-disabled]:not([aria-disabled='false']) {
+  opacity: 0.32;
+  cursor: default;
+}
+
+.charcoal-radio__input {
+  appearance: none;
+  display: block;
+  box-sizing: border-box;
+
+  margin: 0;
+  padding: 6px;
+
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+  border-radius: 999999px;
+  background-color: var(--charcoal-surface1);
+  transition: 0.2s background-color, 0.2s box-shadow;
+}
+.charcoal-radio__input:checked {
+  background-color: var(--charcoal-brand);
+}
+.charcoal-radio__input:checked::after {
+  content: '';
+  display: block;
+  width: 8px;
+  height: 8px;
+  pointer-events: none;
+  background-color: var(--charcoal-text5);
+  border-radius: 999999px;
+  transition: 0.2s background-color, 0.2s box-shadow;
+}
+
+.charcoal-radio__input:not(:checked) {
+  border-width: 2px;
+  border-style: solid;
+  border-color: var(--charcoal-text3);
+}
+.charcoal-radio__input:disabled {
+  cursor: default;
+}
+
+.charcoal-radio__input:not(:disabled):hover {
+  background-color: var(--charcoal-surface1-hover);
+}
+
+.charcoal-radio__input:not(:disabled):active {
+  background-color: var(--charcoal-surface1-press);
+}
+.charcoal-radio__input:not(:disabled):focus {
+  outline: none;
+  box-shadow: 0 0 0 4px rgba(0, 150, 250, 0.32);
+}
+.charcoal-radio__input:not(:disabled):focus-visible {
+  box-shadow: 0 0 0 4px rgba(0, 150, 250, 0.32);
+}
+.charcoal-radio__input:not(:disabled):focus:not(:focus-visible) {
+  box-shadow: none;
+}
+
+.charcoal-radio__input:not(:disabled)[aria-invalid='true'],
+.charcoal-radio__input:not(:disabled)[aria-invalid='true']:focus {
+  box-shadow: 0 0 0 4px rgba(255, 43, 0, 0.32);
+}
+
+.charcoal-radio__input:checked:not(:disabled):hover {
+  background-color: var(--charcoal-brand-hover);
+}
+.charcoal-radio__input:checked:not(:disabled):hover::after {
+  background-color: var(--charcoal-text5-hover);
+}
+.charcoal-radio__input:checked:not(:disabled):active {
+  background-color: var(--charcoal-brand-press);
+}
+.charcoal-radio__input:checked:not(:disabled):active::after {
+  background-color: var(--charcoal-text5-press);
+}
+
+.charcoal-radio__label {
+  font-size: 14px;
+  line-height: 22px;
+  display: flow-root;
+  color: var(--charcoal-text2);
+}
+.charcoal-radio__label::before {
+  display: block;
+  width: 0;
+  height: 0;
+  content: '';
+  margin-top: -4px;
+}
+.charcoal-radio__label::after {
+  display: block;
+  width: 0;
+  height: 0;
+  content: '';
+  margin-bottom: -4px;
+}
```

Comment:
We can remove these styles. The new style doesn't use any hacked adjustments.

Reply:
Oh. Ok!
'll fix the other PRs.